buf/registry/module/v1beta1/commit.proto

```diff
@@ -65,6 +64,9 @@ message Commit {
     (buf.validate.field).required = true,
     (buf.validate.field).string.uuid = true
   ];
-  // VCS commit information.
-  VCSCommit vcs_commit = 7;
+  // The URL for the source control commit that is associated with the BSR Commit's Contents.
+  //
+  // BSR users can navigate to this link to find source control information that is relevant to this BSR Commit
+  // (e.g. commit description, PR discussion, authors, approvers, etc.).
+  string source_control_url = 7 [(buf.validate.field).string.uri = true];
```

Comment:
Do we want to ensure the scheme here is `https` (and `http` if we want to be lenient), or do we need to handle custom schemes depending on the SCM type?

Reply:
Eh, I don't see a need to enforce a particular URL scheme here. The BSR itself doesn't really care what this url is at all really. We are just going to render a link to it that a user can click on. If some company has some use case for deep linking into some native app that is on every developer's machine, I see no reason to stand in their way.